documentation/src/main/asciidoc/topics/dependencies/spring_session_dependencies.adoc

```diff
@@ -0,0 +1,29 @@
+[source,xml,options="nowrap"]
+----
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-spring5-embedded</artifactId>
+            <version>${version.infinispan}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${version.spring}</version>
+        </dependency>
+        <dependency>
+           <groupId>org.springframework</groupId>
+           <artifactId>spring-session-core</artifactId>
```

Comment:
in 9.4 we will have spring 4 and spring 5 too. I did not add the docs yet for the backport I did in the spring 5 PR. 
Concerning this dependency, the value is spring-session for spring 4 and spring-session-core for spring 5